applications/konami/src/konami_util.erl

```diff
@@ -41,3 +43,35 @@ send_break_req(CallId) ->
           ],
     lager:debug("attempting to break ~s", [CallId]),
     kz_amqp_worker:cast(API, fun kapi_metaflow:publish_action/1).
+
```

Comment:
Kazoo applications (those apps under `/applications`) should not call code that lives in other Kazoo applications. If this is accepted, trunkstore would depend on konami being present to avoid crashing.

The proper thing to do would be to put this in an appropriate core library; perhaps `kz_endpoint`?